src/rf/apps/core/models.py

```diff
@@ -222,16 +232,10 @@ class LayerImage(Model):
         default=0,
         help_text='The order which images are layered (starting from 0)'
     )
-    thumb_small = URLField(
-        null=True,
-        blank=True,
-        help_text='80x80 pixels',
-    )
-    thumb_large = URLField(
-        null=True,
-        blank=True,
-        help_text='300x300 pixels',
-    )
+    thumb_small_key = CharField(max_length=255, blank=True, default='',
```

Comment:
Probably don't want the db helptext to be so specific about file size. We might change the size on the front end, if that happens, we shouldn't have to change the database since it only stores a filename.